mmvb_backend/metrics/implementations/correct_conditions_top_n.py

```diff
@@ -0,0 +1,106 @@
+from collections import defaultdict
+
+from django.shortcuts import get_object_or_404
+from django.utils.decorators import classproperty
+
+from benchmarking_sessions.models import BenchmarkingStepStatus
+from cases.models import Case
+from metrics.implementations.base import Metric
+
+
+class CorrectConditionsTop1(Metric):
```

Comment:
That was the initial approach, went with this just to have a cleaner way when automatically listing all metrics. When going with a common base we will need extra mechanisms to avoid (automatically) adding the common base classes to the available metrics list.

Reply:
makes sense! maybe add a comment so that people don't try to refactor it down the road and run into problems? (or know how to fix them at least...)